process/block/metablock.go

```diff
@@ -786,7 +761,7 @@ func (mp *metaProcessor) CreateBlock(
 	return metaHdr, body, nil
 }
 
-func (mp *metaProcessor) isPreviousBlockEpochStart() (uint32, bool) {
+func (mp *baseProcessor) isPreviousBlockEpochStart() (uint32, bool) {
```

Comment:
Please move this func in `process/block/baseProcess.go`

Reply:
Right, moved it